spring-boot-project/spring-boot-docker-compose/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
 	dockerTestRuntimeOnly("io.r2dbc:r2dbc-mssql")
 	dockerTestRuntimeOnly("org.postgresql:postgresql")
 	dockerTestRuntimeOnly("org.postgresql:r2dbc-postgresql")
+	dockerTestRuntimeOnly("com.ibm.db2:jcc")
```

Comment:
Dependencies are meant to be ordered alphabetically. Please move this at the top of the group.